globus_sdk/nexus.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function
+
+from globus_sdk.base import BaseClient
+
+
+class NexusClient(BaseClient):
+    def __init__(self, environment="default"):
+        BaseClient.__init__(self, "nexus", environment)
+
+    def get_goauth_token(self, username, password):
+        """
+        Legacy method for getting a GOAuth token from nexus using
+        globusid username and password. Clients using this should
+        be prepared to migrate to Globus Auth.
+
+        Note that these tokens have a long lifetime and should be saved
+        and re-used.
+        """
+        r = self.get('/goauth/token?grant_type=client_credentials',
```

Comment:
I disagree, I think for short bits of code, single var names are fine and in some ways easier to read.